frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -840,6 +841,8 @@ const QueueItem = ({
                           saveState === "error" && "card-test-input__error",
                           dateBeforeWarnThreshold && "card-correction-input"
                         )}
+                        aria-labelledby="test-date-time-label"
+                        aria-label="test date"
```

Comment:
Honest question, are `aria-label` and `aria-labelledby` meant to be used together on the same element?

https://web.dev/aria-labels-and-relationships/#:~:text=Importantly%2C%20aria%2Dlabelledby%20overrides%20all,labelledby%20label%20always%20takes%20precedence

Reply:
that's a really good question, i am guessing not since aria-labelledby will take precedence